pkg/scheduler/scheduler_test.go

```diff
@@ -766,7 +769,9 @@ func TestSchedulerFailedSchedulingReasons(t *testing.T) {
 	fns := []st.RegisterPluginFunc{
 		st.RegisterQueueSortPlugin(queuesort.Name, queuesort.New),
 		st.RegisterBindPlugin(defaultbinder.Name, defaultbinder.New),
-		st.RegisterPluginAsExtensions(noderesources.FitName, noderesources.NewFit, "Filter", "PreFilter"),
+		st.RegisterPluginAsExtensions(noderesources.FitName, func(plArgs apiruntime.Object, fh framework.Handle) (framework.Plugin, error) {
+			return noderesources.NewFit(plArgs, fh, noderesources.FitWithPodOverhead(utilfeature.DefaultFeatureGate.Enabled(features.PodOverhead)))
```

Comment:
let's not pass a feature if the test doesn't depend on it.

Site note: we should push for graduation of these features next release 😂